workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGen/CodeGeneratorOptions.cs

```diff
@@ -53,6 +56,18 @@ public static CodeGeneratorOptions ParseArguments(ICollection<string> args)
                     "serialization-override=", "OPTIONAL: defines an override for serialization of a single type",
                     u => options.SerializationOverrides.Add(u)
                 },
+                {
+                    "disable-verbose-logging=", "OPTIONAL: disable verbose logging",
+                    v => options.DisableVerboseLogging = string.IsNullOrEmpty(v) || v.Equals("true")
+                },
+                {
+                    "log-to-console=", "OPTIONAL: enable logger output to console",
+                    c => options.EnableLoggingToConsole = string.IsNullOrEmpty(c) || c.Equals("true")
+                },
+                {
+                    "logger-output-dir=", "REQUIRED: absolute path to logger output file",
+                    p => options.AbsoluteLogPath = p
+                },
```

Comment:
just remembered that I want to change these to parse this in a nicer way a la deployment launcher